src/webgpu/shader/execution/expression/call/builtin/fract.spec.ts

```diff
@@ -51,7 +52,7 @@ Component-wise when T is a vector.
 
     const makeCase = (x: number): Case => {
       const result = x - Math.floor(x);
-      if (result < 1.0) {
+      if (result < 1.0 || flushSubnormalNumber(x) === 0.0) {
```

Comment:
`isSubnormalNumber()`?

Reply:
Yeah, we could do, this is just the pattern established in other places. @zoddicus ?

Reply:
I would prefer `isSubnormalNumber()`, other cases doing `flushSubnormalNumber(x)=== 0.0` are probably old code

Reply:
Will update with that change.